src/lib.rs

```diff
@@ -210,6 +212,27 @@ impl<'a> Admonition<'a> {
 </{admonition_block}>"#,
         )
     }
+
+    fn attach_defaults(&mut self, ctx: &PreprocessorContext) -> Result<()> {
+        let table_op = ctx.config.get("preprocessor.admonish.default");
+
+        if let Some(table) = table_op {
+            let toml_defaults: TomlDefaults = toml::from_str(&table.to_string())
```

Comment:
FYI, what you want here is `table.to_owned().try_into().expect(...)`, rather than roundtripping via a string.